Tensile/TensileCreateLibrary.py

```diff
@@ -1291,6 +1291,10 @@ def splitExtraParameters(par):
   for logicFile in logicFiles:
     print1("#   %s" % logicFile)
 
+  # Skip experimental libraries (if exists) when building without lazy loading
+  if not globalParameters["LazyLibraryLoading"]:
+    logicFiles = [f for f in logicFiles if not ("experimental/dtree/" in f or "experimental/grid/" in f)]
+
```

Comment:
could we just exclude all `experimental/` instead of the two specific ones? I think having experimental as a general reserved name is ok, but I'm less a fan of these special cases hardcoded for dtree and grid.

Reply:
actually maybe I'll be more nitpicky and request that the experimental folder name be a Tensile config parameter, that defaults to `experimental/` so it isn't hidden way down here.

Reply:
Added the ExperimentalLogicDir to globalParameters.